### DISCUSSION TITLE
Date & Time - Generic

### DISCUSSION THREAD

**Discussion by esoleyman** [Syriac-Localization/syriac-localization.github.io#3] [General]
We need to review our formatting up and down this section because it doesn't look quite right to me.

**Comment by SargisYonan** [comment]
To do this, we can review https://st.unicode.org/cldr-apps/v#r_datetime/syr//

To me our submissions from last year seem to have addressed most of my big concerns. Let's go over these in more detail just in case

**Reply by esoleyman** [reply]
![Screenshot from 2023-06-23 08-48-55](https://github.com/Syriac-Localization/syriac-localization.github.io/assets/6634387/34824d27-3ae4-409b-95cc-4368122dfd3b)

We are messing up on the **year month day** formatting by putting the month before the day.

**Correct formatting**
13 ܟܢܘܢ ܐܚܪܝܐ 2012 

**Current formatting**
2012 ܟܢܘܢ ܐܚܪܝܐ 13

For some reason this looks wrong in the browser but looks better in my browser. See screenshot.

**Reply by esoleyman** [reply]
Filed bug report [CLDR Jira 16789]( https://unicode-org.atlassian.net/jira/software/c/projects/CLDR/issues/CLDR-16789) to track the inconsistencies between SurveyTool and the DateTime Report.
